### ISSUE TITLE
2 ISSUE TRACKERS, UNCLEAR WHICH ONE IS CANONICAL

### ISSUE THREAD

**Issue by Chealer** [rcaelers/workrave#150]
Workrave currently has 2 general-purpose issues trackers:

1. [This one](https://github.com/rcaelers/workrave/issues)
1. [An older Bugzilla instance](https://issues.workrave.org/)

As it is unclear which one is canonical, it is unclear where users should report and search issues.

This was also [reported on Bugzilla](https://issues.workrave.org/show_bug.cgi?id=1304).

**Comment by mattia-b89** [comment]
Despite official site points you to **2.**,  
I think this one, here, the issue tracker integrated in Github is the new official place where report bugs.

---

_I think_ the truth is: project is almost _sleeping_ (not dead but not actively maintained... there is a plethora of open bugs which no one takes care of, for example...) and when they switched here, it has not been done the migration from old bug tracker (bugzilla) of the already open bugs

**Comment by rcaelers** [comment]
I never deliberately decided to switch from Bugzilla to Github for bug reports. It was enabled and people started to file bugs at Gihub. I prefer the Github bugtracker over the self-hosted Bugzilla, so  I just changed the Bugzilla configuration to no longer accept new bug reports and user registrations. I also added a banner to the Bugzilla bugtracker to redirect to everyone to Github. 

I will keep the Bugzilla tracker alive for a while. The current official site, and also the Workrave crashdump on Windows, point to the older Bugzilla, and there are still a lot of features requests and bug reports in there.

I agree that Workrave is currently not very actively maintained. I'm currently the only developer and my spare time (and motivation) to work on Workrave is limited. Also, I have been working on a number of things that are not very visible (internal code cleanup/refactoring, new docker based build, Qt5 port, macOS port, new website,...).
